crates/ide-assists/src/lib.rs

```diff
@@ -171,6 +170,7 @@ mod handlers {
     mod remove_unused_param;
     mod reorder_fields;
     mod reorder_impl;
+    mod reorder_impl_items;
```

Comment:
Unrelated to the PR, but why assist names differ so much from the label? Wouldn't it be more clear it `reorder_impl` was called `sort_impl_methods_by_trait_definition`? this may be too long, but IMO clearer.

Reply:
Some assist labels have changed over time, and other times it is in fact the very long name I imagine :) Our naming story for assists is not too consistent unfortunately. While I do like descriptive names, in this case I think that name would be too long for my taste at least. There is also the problem with some modules hosting multiple assists (usually their counter parts), so there naming after label wouldn't work anymore either.